docs/embedding/sdk/questions.md

```diff
@@ -139,26 +139,169 @@ To customize the layout, use namespaced components within the `InteractiveQuesti
-| `FilterDropdown`           | A button which has a dropdown for adding, viewing, and editing the filters of the current question                           |
-| `ResetButton`              | The button used to reset the question after the question has been modified with filters/aggregations/etc                     |
-| `Title`                    | The question's title                                                                                                         |
-| `SaveButton`               | Button for saving the question.                                                                                              |
-| `Breakout`                 | A set of badges for adding, viewing, and editing the groupings of the current question                                       |
-| `BreakoutDropdown`         | A button which has a dropdown for adding, viewing, and editing the groupings of the current question                         |
-| `Summarize`                | A set of badges for adding, viewing, and editing the groupings of the current question                                       |
-| `SummarizeDropdown`        | A button which has a dropdown for adding, viewing, and editing the groupings of the current question                         |
-| `Notebook`                 | The Notebook editor that allows for more filter, aggregation, and custom steps                                               |
-| `NotebookButton`           | The button used in the default layout to open the Notebook editor. You can replace this button with your own implementation. |
-| `QuestionVisualization`    | The chart visualization for the question                                                                                     |
-| `QuestionSettings`         | The settings for the current visualization                                                                                   |
-| `QuestionSettingsDropdown` | A settings button that contains the `QuestionSettings` component in the dropdown                                             |
-| `ChartTypeDropdown`        | A dropdown containing the possible visualization options for the current question                                            |
+#### `InteractiveQuestion.BackButton`
+> A navigation button that returns to the previous view. Only renders when `onNavigateBack` is available in context.
 
+Uses [Mantine ActionIcon props](https://v6.mantine.dev/core/action-icon/) under the hood, as well as:
+| Prop | Type | Description |
+|------|------|-------------|
+| className | string | |
+| style | React.CSSProperties | |
+
+#### `InteractiveQuestion.Filter`
+> A set of interactive filter badges that allow adding, editing, and removing filters. Displays current filters as badges with an "Add another filter" option.
+
+| Prop | Type | Description |
+|------|------|-------------|
+| withColumnItemIcon* | boolean | Whether to show column icons in the filter interface |
```

Comment:
what's the rationale behind making a boolean required? I think the argument could be to always expose these options to users so they know these props exist. It just feels a bit usually since boolean props are mostly optional.

Reply:
The actual type seems to be optional actually:

```ts
withColumnItemIcon?: boolean | undefined;
```